### PR TITLE
Fix usage of cwd

### PR DESCRIPTION
**Pull request by StephenWeatherford**
cwd needs to remain the root path. Not sure why this issue only affects docker, but I tested the change with functions and it's fine.

### CODE REVIEW

**Comment by EricJizbaMSFT** [comment]
I think this change is already in master?
https://github.com/Microsoft/vscode-azuretools/blob/master/dev/src/gulp/gulp_webpack.ts

**Comment by EricJizbaMSFT** [comment]
Never mind disregard comment

File: dev/src/gulp/gulp_webpack.ts [hunk 1]
```diff
@@ -15,14 +16,15 @@ export function gulp_webpack(mode: string): cp.ChildProcess {
     env.NODE_OPTIONS = '--max-old-space-size=8192';
 
     return cp.spawn(
-        // https://github.com/nodejs/node-v0.x-archive/issues/2318#issuecomment-249355505
-        process.platform === 'win32' ? 'webpack.cmd' : 'webpack',
+        path.join(
+            './node_modules/.bin/',
```
Comment (EricJizbaMSFT) [inline dev/src/gulp/gulp_webpack.ts]: nit: split up the node_modules and bin in path.join

**Comment by StephenWeatherford** [comment]
Both published.

**Pull request merged**